app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -57,6 +85,11 @@ public Product deleteProduct(Long id) {
         return product;
     }
 
+    /**
+     * 해당 식별자의 상품을 리턴합니다.
+     * @return 상품
+     * @throws ProductNotFoundException 해당 식별자의 상품이 없는 경우
```

Comment:
찾지 못한 것과 없는 것은 다릅니다. 그리고 대부분의 경우 "없다"라고 확실하게 단정할 수 있는 상황은 드물죠.
```suggestion
     * @throws ProductNotFoundException 해당 식별자의 상품을 찾지 못한 경우
```

Reply:
이건 저번에도 이야기해주셨는데 깜빡했네요ㅎㅎ "존재하지 않는" 같은 표현을 "찾지 못한"으로 모두 변경했어요